tests/tests/Data/RawCodeCoverageDataTest.php

```diff
@@ -272,7 +272,6 @@ public function testUseStatementsAreUncovered(): void
             [
                 12,
                 14,
-                15,
```

Comment:
The first element of an array must be counted as executable line

Reply:
in your initial PR - https://github.com/sebastianbergmann/php-code-coverage/pull/892 - you show in the "after" screenshot exactly the oposite - would you be ok to send another PR into this PR branch?

Reply:
You are right: I thought that at least one line of an array declaration should be marked as executable, an in my first PR I set it to be the line of the square bracket.
But that's not true: like you've done here, it's instead correct to count the entire array declaration as non-executable, if it doesn't contain any inside.

Only the variable assignment then is left to be fixed